tests/test-amp-image-dimension-extractor.php

```diff
@@ -54,9 +54,9 @@ function test__normalize_url( $source_url, $expected_url ) {
 class AMP_Image_Dimension_Extractor__By_Downloading__Test extends WP_UnitTestCase {
 
 	function test__valid_image_file() {
-		$source = 'https://i0.wp.com/placehold.it/350x150.png';
+		$source = 'https://placehold.it/350x150.png';
```

Comment:
This [comment](https://github.com/Automattic/amp-wp/pull/480#pullrequestreview-1423633) ("The commit message doesn't match the changes here. The use of Photon seems to be dropped here.") was meant to go here (Still getting used to Github's new review interface :))